overlord/devicestate/devicestate_systems_test.go

```diff
@@ -1706,6 +1707,210 @@ func (s *deviceMgrSystemsCreateSuite) TestDeviceManagerCreateRecoverySystemRemod
 	c.Check(filepath.Join(boot.InitramfsUbuntuSeedDir, "systems", "1234", "snapd-new-file-log"), testutil.FileAbsent)
 }
 
+func (s *deviceMgrSystemsCreateSuite) TestDeviceManagerCreateRecoverySystemRemodelValidationSet(c *C) {
```

Comment:
This test doesn't do a great job of validating much besides making sure that we can fetch the validation set assertions that are in the new model.